cns/api.go

```diff
@@ -22,11 +22,7 @@ const (
 	DeleteNetworkPath             = "/network/delete"
```

Comment:
#should we remove `createnetworkpath` and `deletenetworkpath` too?

Reply:
If you know that they are not being used, we can delete them, but they are not used by CNM so they haven't been removed.